src/current/_includes/releases/v24.1/v24.1.2.md

```diff
@@ -0,0 +1,71 @@
+- Added an error message for unsupported Apache Pulsar [changefeed](https://www.cockroachlabs.com/docs/v24.1/create-changefeed) sink parameters. [#124666][#124666]
+
+<h3 id="v24-1-2-operational-changes">Operational changes</h3>
+
+- Improved [disk usage metric reporting](https://www.cockroachlabs.com/docs/v24.1/ui-cluster-overview-page#capacity-metrics) over volumes that dynamically change their size over the life of the `cockroach` process. [#125050][#125050]
+- The following [cluster settings](https://www.cockroachlabs.com/docs/v24.1/cluster-settings) have had their default values updated from `100000` to `7500`. These settings control the maximum number of [fingerprints](https://www.cockroachlabs.com/docs/v24.1/ui-statements-page#sql-statement-fingerprints) (distinct combinations of statements and [transactions](https://www.cockroachlabs.com/docs/v24.1/transactions)) we can store in memory, with stats being flushed on a `10m` [interval](https://www.cockroachlabs.com/docs/v24.1/interval) by default. Users with workloads producing unique fingerprints higher than this amount within the flush interval may notice that they are missing SQL stats. Users who want to capture those missing stats may increase these settings as necessary. [#125554][#125554]
+  - [`sql.metrics.max_mem_stmt_fingerprints`](https://www.cockroachlabs.com/docs/v24.1/cluster-settings#setting-sql-metrics-max-mem-stmt-fingerprints)
+  - [`sql.metrics.max_mem_txn_fingerprints`](https://www.cockroachlabs.com/docs/v24.1/cluster-settings#setting-sql-metrics-max-mem-txn-fingerprints)
+
+<h3 id="v24-1-2-db-console-changes">[DB Console](https://www.cockroachlabs.com/docs/v24.1/ui-overview) changes</h3>
+
+- The favicon now renders properly for [DB Console](https://www.cockroachlabs.com/docs/v24.1/ui-overview), along with other image files. [#122706][#122706]
+
+<h3 id="v24-1-2-bug-fixes">Bug fixes</h3>
+
+- [`SHOW TYPES`](https://www.cockroachlabs.com/docs/v24.1/show-types) now includes [user defined composite types](https://www.cockroachlabs.com/docs/v24.1/create-type#create-a-composite-data-type). It had omitted those types ever since composite types were added in v23.1. [#124817][#124817]
```

Comment:
```suggestion
- [`SHOW TYPES`](https://www.cockroachlabs.com/docs/v24.1/show-types) now includes [user-defined composite types](https://www.cockroachlabs.com/docs/v24.1/create-type#create-a-composite-data-type). It had omitted those types ever since composite types were added in v23.1. [#124817][#124817]
```

Reply:
FINE THEN RYANNNNNNNN 🤣 

ps thank you :-)